internal/wasm/sdk/opa/config.go

```diff
@@ -30,7 +30,7 @@ func (o *OPA) WithPolicyBytes(policy []byte) *OPA {
 	return o
 }
 
-// WithDataFile configures the JSON data file to load.
+// WithDataFile configures the JSON parsedData file to load.
```

Comment:
nit: looks like `s/data/parsedData/g` got run a bit too eagerly. I know it's super minor but it would be good to fix the comments and the error messages in internal/wasm/sdk/opa/errors.go